map_model/src/map.rs

```diff
@@ -115,21 +115,21 @@ impl Map {
     }
 
     pub fn corrupt_err(path: String, err: Box<dyn std::error::Error>) {
-        println!("\nError loading {}: {}\n", path, err);
+        error!("\nError loading {}: {}\n", path, err);
         if err.to_string().contains("No such file") {
-            println!(
+            error!(
```

Comment:
I had an unrelated issue were the scenarios were failing to load, but I couldn't read the println output in the browser.